src/components/CompareResults/beta/RunValues.tsx

```diff
@@ -23,34 +23,36 @@ const styles = {
 };
 
 function RunValues(props: RunValuesProps) {
-  const { name, mean, runValues: values } = props.revisionRuns;
+  const { name, median, values, stddev, stddevPercent } = props.revisionRuns;
```

Comment:
Any reason we're displaying median and not mean? I'm just curious.

Of course not for this patch: in the future we'll have to display something better than just median or mean, we'll have to find out if the distribution is bimodal / trimodal for example.

Reply:
I believe @gmierz suggested using median instead of mean back when we were starting to develop PerfCompare.